### ISSUE TITLE
Installer does not work on a 32-bit OS

### ISSUE THREAD

**Issue by daviox** [minbrowser/min#2390]
- **Min Version:** 1.31.1
- **Operating system:** Windows 10 Home 22H2 32-bit, build 19045.3930

## Expected Behavior
Min installs successfully.

## Actual Behavior
When trying to install Min browser in 32-bit environment, it fails halfway through.
When checking the logs, it seems that the installer is trying to setup a 64-bit version on a 32-bit OS. The Min directory is present, but trying to run the browser results in `This App Can’t Run on Your PC` error popping-up.

Min browser works fine when using the ia32 release, but it's not integrated as well with the OS as the installed one.

## To Reproduce
Run Min Browser installer in a 32-bit Windows OS

**Comment by PalmerAL** [comment]
Good point, I've added a 32-bit installer: https://github.com/minbrowser/min/tree/windows-32bit-installer

Could you try this build and confirm that it runs?: https://workdrive.zohoexternal.com/file/n7nak3b74b83fcdca41799f44b29e96592934
